Automattic/wp-calypso
# BackupBrowser: Update the header style of the backup browser

Related to p1HpG7-nNV-p2#comment-65834

## Proposed Changes

* Updates the header to keep checkboxes always visible and reduces the number of buttons on the screen to just the headers

## Testing Instructions

* Test locally or via a live branch.  If using a live branch, add ?flags=jetpack/backup-granular to the URL
* Visit the Backup Browser page for a site
* Visual check vs. the linked comment / figma
* Ensure the buttons still function as they do in production



* Before
<img width="705" alt="Screenshot 2023-09-07 at 12 30 28 PM" src="https://github.com/Automattic/wp-calypso/assets/1992658/d61252e9-7ebe-4f8b-a191-1896fc97b2e1">
<img width="708" alt="Screenshot 2023-09-07 at 12 30 35 PM" src="https://github.com/Automattic/wp-calypso/assets/1992658/6b19b23b-477e-4246-bb6b-4c90240cdbff">

* After
<img width="708" alt="Screenshot 2023-09-07 at 12 29 23 PM" src="https://github.com/Automattic/wp-calypso/assets/1992658/fee0be39-6dad-477e-9477-e7c6bc563c00">
<img width="702" alt="Screenshot 2023-09-07 at 12 29 38 PM" src="https://github.com/Automattic/wp-calypso/assets/1992658/314b458d-cec3-4c8d-a25b-e367928e0734">


## Pre-merge Checklist
